report/src/statChart.ts

```diff
@@ -0,0 +1,183 @@
+import { QuartileCandlestick, Scatter } from "@hpcc-js/chart";
+import { format as d3Format, HTMLWidget, Palette } from "@hpcc-js/common";
+
+const siFormatter = d3Format(".2s");
```

Comment:
Might want to use scientific notation for the BellCurve axis `d3Format(".2e")`

Reply:
I am (as I manually set the "ticks")

Reply:
I mean `exponent notation` rather than `SI`. In other words:
20.0e6 instead of 20M
20.0e9 instead of 20G
20.0e12 instead of 20T
20.0e15 instead of 20P

Reply:
Personally I prefer the "s" over the "e" from a readability point of view, I will let @dcamper decide...

Reply:
I am personally comfortable with either, but I think the target audience would prefer SI notation.